### PR TITLE
feature/FOUR-9486: IT5 Default Welcome Screens

### PR DESCRIPTION
**Pull request by pmPaulis**
## Issue & Reproduction Steps
This is a feature defined in the document: https://processmaker.atlassian.net/wiki/spaces/PM4/pages/3123380225/Default+Welcome+Screens

## Solution
Implement the following user stories:
**Home**
- As a Participant, I want quick and convenient access to my personal To-Do list (Open tasks assigned to me) so that I can easily manage and track my tasks.
- As a Participant, I want to have quick and effortless access to My Requests (created by me), allowing me to stay informed about their status and take necessary actions promptly.
- As a Participant, I want to have quick and convenient access to the analytics charts that allows me to access my pinned analytics charts easily, enabling me to monitor key metrics and make informed decisions efficiently.
- As a Participant, I want to have quick and convenient access to the option of making a New Request, allowing me to easily start a Process.

**Designer**
- As a Designer, I want quick and convenient access to create a new asset, enabling me to efficiently design and contribute to the process repository.
- As a Designer I want to have a quick access to Recently Created assets
- As a Designer I want to have a quick access to Recently Opened Projects

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-9486

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

## Environment Fall
ci:deploy
ci:next
ci:screen-builder:feature/FOUR-9486
ci:package-projects:feature/FOUR-9486

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-0e373669aa.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-0e373669aa.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-0e373669aa.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-0e373669aa.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-0e373669aa.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-0e373669aa.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Review by luNunezProcessmaker — APPROVED** [review]

File: resources/js/processes/designer/AssetButtons.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    asset_name_all: String,
+    urlPath: String,
+    urlAsset: String,
+    asset_name: String,
+    asset_name_new: String,
+  },
+  data() {
+    return {
+      countCategories: 2,
+    };
```
Comment (CarliPinell) [inline resources/js/processes/designer/AssetButtons.vue:40]: please remove unused variable:  countCategories: 2

**Review by CarliPinell — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-0e373669aa.eng.processmaker.net

**Pull request merged**